bokehjs/test/models/ranges/data_range1d.coffee

```diff
@@ -225,11 +231,20 @@ describe "datarange1d module", ->
       [a, b] = r._compute_range(0.01, 10)
       expect(a).to.be.closeTo(0.0017782794100389264, 1e-12)
       expect(b).to.be.closeTo(56.23413251903488, 1e-12)
+      r.range_padding = 0
+      [a, b] = r._compute_range(0.01, 10)
+      expect(a).to.be.closeTo(0.01, 1e-12)
+      expect(b).to.be.closeTo(10, 1e-12)
```

Comment:
Nice to see this, we probably need to use `closeTo` in a variety of places elsewhere, I somehow missed that it existed until now.